File: examples/weather-api-go/_generators/endpoint/register_route.go
---
to: main.go
inject: true
after: "// ROUTES"
desc: Register the new endpoint route
notes: "The new /{{ name | kebab }} endpoint is ready. Restart the server to use it."
---
	mux.HandleFunc("GET /{{ name | kebab }}/{city}", {{ name | camel }}Handler)

File: examples/weather-api-python/_generators/endpoint/register_route.py
---
to: app.py
inject: true
after: "# ROUTES"
desc: Register the new endpoint route
notes: "Remember to import the handler in app.py if needed."
---

@app.route("/{{ name | kebab }}/<city>")
def {{ name | snake }}(city: str):
    """Return {{ name | snake }} data as JSON."""
    from {{ name | snake }} import {{ name | snake }}_handler
    return {{ name | snake }}_handler(city)
